src/utils.js

```diff
@@ -0,0 +1,41 @@
+const cleanAllMoviesData = data => {
```

Comment:
Amazing way to clean up our logic code and put it in one place before using the data elsewhere - reduced repetition and allowed us to find any errors quicker